frontend/src/store.ts

```diff
@@ -0,0 +1,28 @@
+// import { applyMiddleware, createStore, compose } from 'redux'
+import { configureStore } from '@reduxjs/toolkit'
+import { createLogger } from 'redux-logger'
+import rootReducer from './reducers'
+
+const logger = createLogger({
+	level: 'info'
+})
+
+const store = configureStore({
+	reducer: rootReducer,
+	// Disable immutable check because it throws an exception if withItem() tries to modify
```

Comment:
I would recommend creating tickets (in the backlog) for those or we'll forget to fix these issues.